client/src/sass/main.scss

```diff
@@ -0,0 +1,16 @@
+$color-yellow: #FFC911;
```

Comment:
consider giving these more meaningful names e.g. $main-yellow or $background-yellow as we are likely to end up using more than one shade of yellow etc. probably okay for now though?